README.md

```diff
@@ -1,14 +1,14 @@
 # Wowser Math
 
-[![Join Community](https://img.shields.io/badge/discord-join_community-blue.svg?style=flat)](https://discord.gg/DeVVKVg)
-[![Version](https://img.shields.io/npm/v/@wowserhq/math.svg?style=flat)](https://www.npmjs.org/package/@wowserhq/math)
+[![Join Community](https://badgen.net/badge/discord/join%20community/blue)](https://discord.gg/DeVVKVg)
+[![Version](https://badgen.net/npm/v/@wowserhq/math)](https://www.npmjs.org/package/@wowserhq/math)
+![Node Version](https://badgen.net/badge/node/10+/green)
```

Comment:
Let's drop `Node Version`, since the shipped library compat is mentioned in the README, and includes browsers as well as Node. `math` can be used in either env.

Reply:
I did not see the Node version mentioned all the way in the bottom, will get rid of the badge 👍